code/FinanceManager.Components/Components/FinancialAccounts/CurrencyAccountComponents/AccountHistoryToolbar.razor.css

```
@@ -0,0 +1,11 @@
+/* Mute outlined buttons/toggles so they match the text-field border */
+::deep .mud-button-root.mud-button-outlined-default,
```

Stylelint (17.12.0)

[error] Unknown pseudo-element selector "::deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)